modules/brainbrowser/php/brainbrowser.class.inc

```diff
@@ -22,18 +22,8 @@ namespace LORIS\brainbrowser;
  * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
  * @link     https://www.github.com/aces/Loris/
  */
-class Brainbrowser extends \NDB_Form
+class Brainbrowser extends \NDB_Page
```

Comment:
not sure why this file is in the pull request?

Reply:
Huh? I am surprised to see that too...I didn't edit or commit this file. My commit seems to have "herited" changes that were done by Xavier on this file.